src/tile.js

```diff
@@ -53,7 +54,7 @@
  * drawing operation, in pixels. Note that this only works when drawing with canvas; when drawing
  * with HTML the entire tile is always used.
  */
-$.Tile = function(level, x, y, bounds, exists, url, context2D, loadWithAjax, ajaxHeaders, sourceBounds) {
+$.Tile = function(level, x, y, bounds, exists, url, postData, context2D, loadWithAjax, ajaxHeaders, sourceBounds) {
```

Comment:
`postData` should go at the end of the parameter list to maintain backwards compatibility (in case anyone is creating tiles externally in their app).

Reply:
Yes, I was thinking about logically consistent positions in groups of arguments, not about the backward compatibility.